### DISCUSSION TITLE
create file after choosing folder

### DISCUSSION THREAD

**Discussion by imed-ghomari** [SilentVoid13/Templater#592] [Templates Showcase]
```
<%*
const folders = this.app.vault.getAllLoadedFiles().filter(i => i.children).map(folder => folder.path);
const folderChoicePath = await tp.system.suggester(folders, folders);
const folder = app.vault.getAbstractFileByPath(folderChoicePath)
await tp.file.create_new("","",true,folder).basename
_%>
```

**Comment by edhilgendorf** [comment]
Is this still working? I get infinite popups, but this is a great idea, thank you!

**Reply by mnowotnik** [reply]
@edhilgendorf I created a plugin based on the need of this feature, check it out:
https://github.com/mnowotnik/obsidian-user-plugins
It's probably not yet available in Community Plugins

**Reply by imed-ghomari** [reply]
Sorry for the late reply @edhilgendorf, yes, it's working fine for me (templater 1.12.0). And thank you, @mnowotnik, for the suggestion and the plugin.
